apps/condo/domains/notification/tasks/deliverMessage.js

```diff
@@ -46,6 +53,8 @@ const MESSAGE_SENDING_STATUSES = {
     [MESSAGE_RESENDING_STATUS]: true,
 }
 
+const throttlingCacheClient = getRedisClient('NewsItemCommonHooksSet', 'throttleCommonNewsItemsNotifications')
```

Comment:
notification domain should know nothing about any other domain, including news. I think this should be moved to ADR7 settings and made an abstract mechanism for throttling any type of notificxation if needed. Also please update ADR-7 docs.

Reply:
Client name was fixed